src/pipelines/openai.py

```
@@ -789,36 +981,79 @@ def _compose_options(self, runtime_options: Optional[Dict[str, Any]]) -> Dict[st
+        if not audio_bytes:
+            return b"", int(merged["target_format"]["sample_rate"])
+
+        fmt = (merged.get("response_format") or "wav").lower()
+        if fmt == "wav":
+            if not (audio_bytes[:4] == b"RIFF" and b"WAVE" in audio_bytes[:32]):
+                header = audio_bytes[:12]
+                raise RuntimeError(
+                    "Failed to decode OpenAI WAV payload: expected RIFF/WAVE header "
+                    f"but received bytes starting with {header!r}. Ensure OpenAI TTS `response_format` is set to 'wav' "
+                    "and is being sent as `response_format` in the request."
+                )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)